pkg/v1/tkg/clusterclient/clusterclient.go

```diff
@@ -242,6 +242,8 @@ type Client interface {
 	PatchClusterWithOperationStartedStatus(clusterName, namespace, operationType string, timeout time.Duration) error
 	// PatchClusterObjectWithTKGVersion applies patch to cluster objects based on given tkgVersion string
 	PatchClusterObjectWithTKGVersion(clusterName, clusterNamespace, tkgVersion string) error
+	// PatchClusterObjectAnnotations applies patch to cluster objects to update annotation with specified key:value
+	PatchClusterObjectAnnotations(clusterName, namespace, key, value string) error
```

Comment:
I like the fact that you added this function! It makes the code that much more readable. Good stuff!